probes/archived/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: archived
+short: Check that the project is archvied
+motivation: >
+  A project which is not active might not be patched, have its dependencies patched, or be actively tested and used. However, a lack of active maintenance is not necessarily always a problem. Some software, especially smaller utility functions, does not normally need to be maintained. For example, a library that determines if an integer is even would not normally need maintenance unless an underlying implementation language definition changed. A lack of active maintenance should signal that potential users should investigate further to judge the situation.
+implementation: >
+  The probe checks the Archived Status of a project.
+outcome:
+  - If the project is archived, the outcome is OutcomeNegative.
+  - If the project is not archived, the outcome is OutcomePositive.
+remediation:
```

Comment:
Just re-iterating the confusion you mention in the PR body. Is it worth framing the probe as `notArchived` so positive and negative make sense? @laurentsimon

Reply:
I thought about that; AFAIR, my main concern was that we would end up with a double-negative: OutcomeNegative = "No the project is not archived"

Reply:
But I do agree on the confusion from the current setup that the probe checks whether the project is archived, and OutcomeNegative means "Yes".

Reply:
Renamed.